_includes/misc/advcode-shortcuts.md

```diff
@@ -0,0 +1,13 @@
+## Advanced Code Editor search and replace keyboard shortcuts
+
+Advanced Code Editor provides the following shortcuts for search and replace within the code dialog.
+
+| Action | PC | Mac |
```

Comment:
Should `Cmd` be `Command` here? as that's how it's documented on the keyboard shortcuts page.

Reply:
:facepalm:

Reply:
`Cmd` is more of the standard within Apple circles but if it's consistently `Command` elsewhere in our documentation then that's cool.

Reply:
Fixed in https://github.com/tinymce/tinymce-docs/pull/1594/commits/42b7fcf5c90819e237ab7dccd63ff44883971ee0